packages/web-components/fast-components/src/styles/patterns/button.ts

```diff
@@ -92,6 +92,15 @@ export const BaseButtonStyles = css`
         border: 0;
     }
 
+    .control:disabled span
```

Comment:
Thanks for the PR. Curious thought, this only applies to `span`?

Reply:
The button template for the fast-button is defined as:
```
#shadow-root
<button>
  <span part='start></span>
  <span class='content'><slot></slot></span>
  <span part='end'></span>
</button>
```
so the styles are specific to the spans here

Reply:
Ok, I think I misunderstood the problem you were trying to solve based on the description you provided.